btr-web/btr-main-app/lang/en.json

```diff
@@ -118,7 +119,14 @@
     },
     "taxNumber": "A 9-digit Social Insurance Number (SIN), an Individual Tax Number (ITN), or a Temporary Taxation Number (TTN) is required if this person has one.",
     "taxResidency": "Indicate if this person is a resident of Canada for Income Tax purposes.",
-    "reviewConfirm": "Review the information in your registration. If you need to change anything, return to the previous step to make the necessary change."
+    "reviewConfirm": "Review the information in your registration. If you need to change anything, return to the previous step to make the necessary change.",
+    "certify": {
+      "certification": "Certification",
+      "part1": "I, ",
+      "part2": ", certify that I have relevant knowledge of this BC Benefit Company and that I am authorized to make this filing.",
```

Comment:
This should be one section because the 'I, ' might be placed in a different part of the sentence / might not be there at all depending on the language. You can interpolate things like the name inside here (I did this with the table text if you want an example)

Reply:
resolving in future ticket